### ISSUE TITLE
hi there is some error compiling serominer in linux ubuntu

### ISSUE THREAD

**Issue by wolfdrew** [sero-cash/serominer#1]
i am trying to compile and build using cmake and somehow it stop and give me this error. is there any solution to solve the problem?

root@vmi270460:/home/sero/serominer-0.3.0# cmake .
-- [cable ] Cable 0.2.14 initialized
-- [cable ] Build type: Release
Boost find config
-- Boost version: 1.58.0
-- Found the following Boost libraries:
--   system
--   filesystem
--   thread

-------------------------------------------------------------------
-- CMake 3.5.1
-- Build Release / Linux
-------------------------------------------------------------------
-- OPENCL         Build OpenCL mining components               OFF
-- CUDA           Build CUDA mining components                 ON
-- CPU            Build CPU mining components                  ON
-- DBUS           Build D-Bus components                       OFF
-- API            Build API Server components                  ON
-- DEVBUILD       Build with developer logging                 OFF
-------------------------------------------------------------------

CMake Warning at libethash/cmake/cable/bootstrap.cmake:45 (message):
  [cable ] Cable 0.2.14 (version older than 0.2.16) initialized in the
  `serominer` parent project
Call Stack (most recent call first):
  libethash/CMakeLists.txt:11 (include)

CMake Error at serominer/CMakeLists.txt:19 (find_package):
  Could not find a package configuration file provided by "CLI11" with any of
  the following names:

    CLI11Config.cmake
    cli11-config.cmake

  Add the installation prefix of "CLI11" to CMAKE_PREFIX_PATH or set
  "CLI11_DIR" to a directory containing one of the above files.  If "CLI11"
  provides a separate development package or SDK, be sure it has been
  installed.

-- Configuring incomplete, errors occurred!
See also "/home/sero/serominer-0.3.0/CMakeFiles/CMakeOutput.log".
See also "/home/sero/serominer-0.3.0/CMakeFiles/CMakeError.log".
root@vmi270460:/home/sero/serominer-0.3.0#

any help please?

**Comment by toptim** [comment]
same error
ubuntu 16 cuda 9.2 driver 396.82

**Comment by toptim** [comment]
fix folder
~/serominer/libethcore
help me

**Comment by wolfdrew** [comment]
Hi toptim. What do you mean? Should i download new source code release which has new fix in /serominer/libethcore ?

**Comment by toptim** [comment]
no, just check that local folder the same of the folder on github
